### PR TITLE
Create how-to-win-friends.md

### PR DESCRIPTION
**Pull request by pokethan**

### CODE REVIEW

File: how-to-win-friends.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Book Report
+
+## Title
+
+how to win friends and influence people.
+
+## Author(s)
+DALE CARNEGIE.
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: There should be a linebreak above this line.

File: how-to-win-friends.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## What are the main characters in the story?
+does not apply
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: The letter "D" in "does" should be capitalized.

File: how-to-win-friends.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## What is the setting of the story? (Where and when does it take place?)
+does not apply
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: The letter "D" in "does" should be capitalized.

File: how-to-win-friends.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## What are the main events/ideas from the story
+the principles.
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: There should be a line break above this line.

Comment (douglasjarquin) [inline how-to-win-friends.md]: The letter "T" in "the" should be capitalized.

File: how-to-win-friends.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+principle 1:Begin with praise and honest appreciation.
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: The letter "P" in "principle" should be capitalized.

Comment (douglasjarquin) [inline how-to-win-friends.md]: There should be a space after each ":".

File: how-to-win-friends.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+principle 2:Call attention  to people's mistakes indirectly.  
+principle 3:Talk about your mistakes before criticizing the other person.
+principle 4:Ask questions instead of giving direct orders.
+principle 5:Let the other person save face. 
+principle 6:Praise the slightest improvement and praise every enviorment.Be "hearty in your praise.  
+principle 7:Give the other person a fine reputation to live up to. 
+principle 8:Use encouragement.Make fault easy to correct.
+principle 9:Make the other person happy about doing the thing you suggest.
+ If someone reads the principles then i hope you can apply these in your life!
+## How does the story end?
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: There should be a linebreak above this line.

File: how-to-win-friends.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+does not apply
```
Comment (douglasjarquin) [inline how-to-win-friends.md]: There should be a linebreak above this line.

Comment (douglasjarquin) [inline how-to-win-friends.md]: The letter "D" in "does" should be capitalized.

**Review by douglasjarquin — CHANGES_REQUESTED** [review]

File: how-to-win-friends.md (hunk 6 above)
Comment (pokethan) [inline how-to-win-friends.md:32]: is it good now?

File: how-to-win-friends.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# Book Report
+
+## Title
+
+how to win friends and influence people.
```
Comment (douglasjarquin) [inline how-to-win-friends.md:5]: The title should be written like "How to Win Friends & Influence People". Got that from amazon.com.

File: how-to-win-friends.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+## Author(s)
+
+DALE CARNEGIE.
```
Comment (douglasjarquin) [inline how-to-win-friends.md:9]: Only the "D" and "C" should be capitalized here.

File: how-to-win-friends.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## What are the main characters in the story?
+Does not apply
```
Comment (douglasjarquin) [inline how-to-win-friends.md:13]: Try line break again. Should be above.

File: how-to-win-friends.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## What is the setting of the story? (Where and when does it take place?)
+Does not apply
+
+
+## What are the main events/ideas from the story
+the principles.
```
Comment (douglasjarquin) [inline how-to-win-friends.md:21]: Should be written as:

> The Principles:

With a line break below.

File: how-to-win-friends.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+principle 1:Begin with praise and honest appreciation.
```
Comment (douglasjarquin) [inline how-to-win-friends.md:22]: Capitalize the "P," and put a space after the ":"

**Pull request merged**